src/components/Header.js

```diff
@@ -10,7 +11,7 @@ const Header = () => {
         <div className="header-container">
             <div className="team">
                 <div className="logo-container">
-                    <img src="https://cdn.nba.net/assets/logos/teams/secondary/web/TOR.svg" height="28" title="raptors logo" alt="raptors logo"></img>
+                    <img src={raptorsLogo} title="raptors logo" alt="raptors logo"></img>
```

Comment:
Okay, so this entire section should have the `<img>` element inside of the `<button>` one. Let's change the code for this jsx as well as the relative scss.

Firstly, Let's restructure this html to look like so:

```
        <div className="header-container">
            <div className="team">
                <button>
                    <img src={raptorsLogo} title="raptors logo" alt="raptors logo"/>
                    <span>Toronto Raptors</span>
                    <i className="drop-down-team"/>
                </button>
            </div>

            ...
```

NOTE: When you have an empty element like `<i className="drop-down-team"></i>` it should always be written as `<i className="drop-down-team"/>`

And then in App.scss replace all the `.team` css with the code below:

```
  .team {
    position: relative;
    display: flex;
    background: $background-color-purple;

    button {
      display: flex;
      align-items: center;
      padding: 0 15px 0 10px;
      height: 100%;
      border: none;
      background: none;
      color: $white;
      font-family: spurs;
      font-size: 22px;
      letter-spacing: 1px;
      text-transform: uppercase;
      cursor: pointer;

      img {
        height: 35px;
      }

      span {
        margin: 2px 13px 0 8px;
      }
    }

    &::after {
      content: '';
      position: absolute;
      left: 100%;
      z-index: 1;
      border-top: 45px solid $background-color-purple;
      border-right: 16px solid transparent;
    }
  }
```

ANOTHER NOTE: As you've noticed, the sketch designs don't always translate the correct dimensions for items. In this case, I've had to really eyeball and tweak pixel by pixel till it looks correct on the screen. The lesson here being, that a front end dev should alway have a designers mindset.

Reply:
Cool. This is different from what I've heard at the other place I worked. Their view was the dev is on a tight timeline and beholden to the client, and that designer will send you stuff and it's up to you to balance how long it will take with whether a particular designer element is worth the time. But your point is taken and that technique you showed me yesterday mapping the sketch file to the browser window will help me match the design quickly and accurately.

Reply:
The other place you worked at was wrong, lol. There is always a battle between timeline and accuracy no doubt, but a good front end dev will be able to do both. Not that I don't appreciate the point you brought up. I just wish your last job spent more time on accuracy and scalability than on renaming Chinese girls and smoking weed binge watching mediocre sci fi comedy series.